test/typing/plain_files/orm/session.py

```diff
@@ -97,6 +98,12 @@ class Address(Base):
         User.id
     ).offset(User.id)
 
+    # test #11083
+
+    with sess.begin() as tx:
+        # EXPECTED_TYPE: SessionTransaction
+        reveal_type(tx)
```

Comment:
I am not sure how to properly write a type-checker test; or perhaps it isn't running. If I revert `until.py` to the version before this PR this test does not fail.